aspnetcore/fundamentals/websockets.md

```diff
@@ -4,6 +4,7 @@ author: tdykstra
 description: Learn how to get started with WebSockets in ASP.NET Core.
 manager: wpickett
 ms.author: tdykstra
+ms.custom: mvc
 ms.date: 03/25/2017
```

Comment:
Please update the date. The changes were significant enough to warrant the update.